Makefile

```diff
@@ -92,6 +92,14 @@ compile-json:
 	php -dopcache.enable_cli=1 build/compile-json.php
 	git diff --name-only | grep ^src/data/.*\.json\.php$ || true
 
+annotate-clients: clean
+	php -dopcache.enable_cli=1 build/annotate-clients.php --tag=latest
```

Comment:
That enables fast file linting with `opcache_compile`. Unfortunately, it turns out that you cannot lint code declaring symbols in this way, so I can remove that parameter.